app/controllers/trips_controller.rb

```diff
@@ -2,4 +2,8 @@ class TripsController < ApplicationController
   def index
     @trips = Trip.all.paginate(page: params[:page], per_page: 30)
```

Comment:
Is that really all there is to pagination? It has to be more complex than this...

Reply:
That's really all there is! You also have to added <%= will_paginate @trips %> to the index page so you can scroll between pages.